ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml

```diff
@@ -18,12 +18,53 @@ periodics:
     labels:
       master: ci.openshift.redhat.com
     name: azure_build_node_image_centos
-- agent: jenkins
+- agent: kubernetes
   cron: ""
+  decorate: true
   interval: 24h
   labels:
     master: ci.openshift.redhat.com
   name: azure_build_base_image_rhel
+  skip_cloning: true
+  spec:
+    containers:
+    - args:
+      - --artifact-dir=$(ARTIFACTS)
+      - --secret-dir=/usr/local/e2e-azure-secret
+      - --template=/usr/local/build-base-image
+      - --target=build-base-image
+      command:
+      - ci-operator
+      env:
+      - name: CONFIG_SPEC
+        valueFrom:
+          configMapKeyRef:
+            key: release-3.10.yaml
```

Comment:
Hrm, interesting. The base image is not tied to any particular openshift version but since we keep the playbook inside openshift-ansible which follows origin branching we have to use something. I think this is fine but we may want to just use master.yaml if we don't want to ensure fixes in the base playbook have to be backported to release-3.10. Hope that makes sense :)

Reply:
I know it is not required but I highly recommend that we use the playbooks that match the node build image.  Consistency, avoid confusion, and for all of our sanity.

Reply:
ok leaving this as is